package.json

```
@@ -2864,6 +2864,150 @@
 												]
 											}
 										]
+									},
+									{
+										"key": "languageId:python",
+										"name": "Python",
+										"type": "bindings",
+										"bindings": [
+											{
+												"key": "=",
+												"name": "+Format",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "=",
+														"name": "Format document",
+														"type": "command",
+														"command": "editor.action.formatDocument"
+													},
+													{
+														"key": "d",
+														"name": "+Format document with",
```

```suggestion
														"name": "Format document",
```
It seems like this is from the go major mode, I will fix the wording there as well

---

Maybe this could be totally removed, since it's the same of `spc m =`

---

Or maybe he/she wanted to put another action here.

---

I removed it on my side as I could not find it in the Python layer doc of spacemacs.

---

I have seen this in a few places [[1](https://github.com/VSpaceCode/VSpaceCode/pull/159/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R2938-R2944)][[2](https://github.com/VSpaceCode/VSpaceCode/blob/v0.8.5/package.json#L2126-L2131)] and I think I figure it out why it is name `d` it's probably means document.

That makes me rethink about the whole `=` format in our bindings. I am thinking we should have unified view on the format commands, and opened an issue https://github.com/VSpaceCode/VSpaceCode/issues/160. Since we have multiple major modes PR in flights. I suggest we merge them and fix it as part of https://github.com/VSpaceCode/VSpaceCode/issues/160.

---

I will mark this resolve and will fix them as part of #160.